ui/src/chat/ChatScroller/ChatScroller.tsx

```diff
@@ -151,68 +151,6 @@ export default function ChatScroller({
     [messages, replying]
   );
 
-  const mess = useMemo(
-    () =>
-      keys.reduce(
-        (acc, val) => acc.set(val, messages.get(val)),
-        new BigIntOrderedMap<ChatWrit>()
-      ),
-    [keys, messages]
-  );
-
-  const [indexData, setIndexData] = useState<{
-    firstItemIndex: number;
-    data: bigInt.BigInteger[];
-  }>({
-    firstItemIndex: FIRST_INDEX,
-    data: keys,
-  });
```

Comment:
unfortunately we can't get rid of this. if you scroll back on any channel where it has to load the old posts, you'll see it start to load and then completely lose your position because of the way virtuoso does indexes.

This was their official suggestion: https://virtuoso.dev/prepend-items/